ETH breakout

hieuh25 opened the pull request:
- Added a notebook for the same strategy running on Quickswap pair, with overriden fee 5bps
- Added another notebook for adding credit supply positions using Aave v2 lending data

miohtama commented:
Can you add a short comment at the top of the notebook 
- What is the filename of the original notebook it is based of
- What was changed

hieuh25 commented:
> Can you add a short comment at the top of the notebook
> 
> * What is the filename of the original notebook it is based of
> * What was changed

I added the description and also fixed it for other notebooks in the same folder
